sample/src/main/java/com/karumi/dexter/sample/ApplicationSampleActivity.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2016 Karumi.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.karumi.dexter.sample;
+
+import android.app.Activity;
+import android.os.Bundle;
+import butterknife.ButterKnife;
+
+/**
+ * Activity to show if Dexter works when have been invoked from the application.
+ */
+public class ApplicationSampleActivity extends Activity {
+
+  @Override protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.application_sample_activity);
+    ButterKnife.bind(this);
```

Comment:
I don't understand the usage of this Activity. Can you provide us more information?